packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -100,6 +100,26 @@ function getFrameworkTests(opts: {
 				},
 				flags: ["--style", "sass"],
 			},
+			react: {
+				promptHandlers: [
+					{
+						matcher: /Select a variant:/,
+						input: [keys.enter],
+					},
+				],
+				unsupportedOSs: ["win32"],
```

Comment:
The current `create-vite` template doesn't pass on Windows either, so this is not a regression. But at some point we should look into fixing that.